docs/ug/configFiles.md

```diff
@@ -63,7 +63,7 @@ Optionally, you can use an `author-config.csv` (which should be in the same dire
 |-------------|-------------|
 | Repository's Location | Same as `repo-config.csv`. Default: all the repos in `repo-config.csv` |
 | Branch | The branch to analyze for this author, e.g., `master`. Default: the author will be bound to all the repos in `repo-config.csv` that has the same repo's location, regardless of branch. |
-| Author's Git Host ID {{ mandatory }} | Username of the target author's profile on GitHub, GitLab or Bitbucket, e.g.`JohnDoe`. |  
+| Author's GitHub ID {{ mandatory }} | Username of the target author's profile on GitHubHub, GitHubLab or Bitbucket, e.g.`JohnDoe`. |  
```

Comment:
The `Hub` is repeated

Reply:
Oh crap, good spot. I was trying to use line replacement to avoid intellij auto expanding the tables